### DISCUSSION TITLE
Acontecimentos Marcantes #1

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#496] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/acontecimentos-marcantes-1/

**Comment by the-coffee-monk** [comment]
![null](<https://i.imgur.com/PChduQI.gif>)

**Comment by angellisy** [comment]
Tomara que dê certo uma coletânea dessas :heart\_eyes:

**Comment by maryon-telvanni** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/videos/acontecimentos-marcantes-1/#comment_1173954501>):

> Tomara que dê certo uma coletânea dessas 😍

Tomara mesmo. Existem MUITOS outros momentos de certa forma "menores" que também valem a pena ser lembrados. Não só coisas como 14-bis, Homem na Lua, Guerras Mundiais, Chernobyl, 11/09, etc. que na maioria das vezes já têm seus lugares nos livros de História.

**Comment by vish-blog** [comment]
> [Maryon Telvanni](</perfil/?user=maryon-telvanni>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/videos/acontecimentos-marcantes-1/#comment_1174081452>):

> Tomara mesmo. Existem MUITOS outros momentos de certa forma "menores" que também valem a pena ser lembrados. Não só coisas como 14-bis, Homem na Lua, Guerras Mundiais, Chernobyl, 11/09, etc. que na maioria das vezes já têm seus lugares nos livros de História.

Podem ter certeza que em recebendo material, assim que possível, eles serão postados
